frontend/packages/kubevirt-plugin/integration-tests-cypress/cypress.json

```diff
@@ -3,6 +3,8 @@
   "screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots",
   "videosFolder": "../../../gui_test_screenshots/cypress/videos",
   "video": true,
+  "videoCompression": false,
```

Comment:
Why disable video compression?

Reply:
to skip processing time for the video, it could reduce the entire execution time especially when the test code base get larger and larger.

Reply:
Are we worried about running the tests locally or execution time in CI? I suspect this won't make a tremendous difference in execution time in CI, and the storage saved could be significant.

Reply:
I won't run the entire tests locally, so I'm considering to reduce execution time in CI, we also run it in downstream Jenkins CI.
Storage save is already significant if the function deleting video on PASS work here, so I want to skip the processing time.

Reply:
By looking at the build logs, the processing time of a PASS spec is onyl a few seconds (around 5s), so I think it's okay to drop off this feature.